controller/get.controller.js

```diff
@@ -16,9 +16,27 @@ const sendAllArticles = (request, response, next) => {
     selectAllArticles().then((result) => response.status(200).send(result))
 }
 
+const sendComments = (request, response, next) => {
+    const { articleId } = request.params;
+    selectArticleId(articleId)
+        .then(() => {
+            selectComments(articleId)
+                .then((result) => {
+                    response.status(200).send(result);
+                })
+                .catch((error) => {
```

Comment:
We should only need one catch block in our entire promise chain right at the end, this will catch anything that goes wrong throughout the whole process :)